README.md

```diff
@@ -1,56 +1,32 @@
 
-The Iterative Provider makes it easy to:
+- **Provision Resources**: create cloud compute & storage resources without reading pages of documentation
+- **Sync & Execute**: easily sync & run local data & code in the cloud
+- **Low cost**: transparent auto-recovery from interrupted low-cost spot/preemptible instances
+- **No waste**: auto-cleanup unused resources (terminate compute instances upon job completion/failure & remove storage upon download of results)
+- **No lock-in**: switch between several cloud vendors with ease due to concise unified configuration
 
-- Rapidly move local machine learning experiments to a cloud infrastructure
-- Take advantage of training models on spot instances without losing any progress
-- Unify configuration of various cloud compute providers
-- Automatically destroy unused cloud resources (compute instances are terminated on job completion/failure, and storage is removed when results are downloaded)
+Supported cloud vendors include:
 
-The Iterative Provider can provision resources with the following cloud providers and orchestrators:
-
-- Amazon Web Services
+- Amazon Web Services (AWS)
 - Microsoft Azure
-- Google Cloud Platform
-- Kubernetes
-
-## Documentation
-
-See the [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) guide to learn how to use the Iterative Provider. More details on configuring and using the Iterative Provider are in the [documentation](https://registry.terraform.io/providers/iterative/iterative/latest/docs).
-
-## Support
-
-Have a feature request or found a bug? Let us know via [GitHub issues](https://github.com/iterative/terraform-provider-iterative/issues). Have questions? Join our [community on Discord](https://discord.gg/bzA6uY7); we'll be happy to help you get started!
+- Google Cloud Platform (GCP)
+- Kubernetes (K8s)
 
-## License
+## Usage
 
-Iterative Provider is released under the [Apache 2.0 License](https://github.com/iterative/terraform-provider-iterative/blob/master/LICENSE).
+See the [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) guide for a more detailed guide.
 
-## Development
-
-### Install Go 1.17+
-
-Refer to the [official documentation](https://golang.org/doc/install) for specific instructions.
-
-### Clone the repository
-
-```console
-git clone https://github.com/iterative/terraform-provider-iterative
-cd terraform-provider-iterative
-```
+### Requirements
 
-### Install the provider
-
-Build the provider and install the resulting binary to the [local mirror directory](https://www.terraform.io/docs/cli/config/config-file.html#implied-local-mirror-directories):
-
-```console
-make install
-```
+- [Install Terraform 1.0 or greater](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform)
```

Comment:
It might be better to mention a command instead of an external page link.

This might be a good opportunity to check version of terraform (and get rid of --upgrage option in `terraform init` - see below)

Reply:
😅 Installing Terraform does require a whole page; instructions vary greatly between operating systems.

I don't know if linking to the official documentation will deter any users, but it's probably better than maintaining an outdated duplicate.

Reply:
As stared below, `--upgrade` is not needed unless you (re)compile the provider.

Reply:
Yes, these are good reasons to keep the link to it. The link should stay. But at the end of the day I need to visit an external page just to copy-past `brew install terraform`.

I was hoping that we can provide shortcuts for Mac and Linux users. 

Also, we lose some users who click to external link. Especially the once who is not familiar with Terraform.

Reply:
Makes sense. Added one-liners for most cases.